picard/browser/browser.py

```diff
@@ -107,16 +112,26 @@ def start(self):
         if self.server:
             self.stop()
 
+        if BROWSER_INTEGRATION_IPV6:
+            LISTEN_ALL = '::'
+            ADDRESS_FAMILY = socket.AF_INET6
+        else:
+            LISTEN_ALL = '0.0.0.0'
+            ADDRESS_FAMILY = socket.AF_INET
+
+        class OurHTTPServer(ThreadingHTTPServer):
+            address_family = ADDRESS_FAMILY
+
         config = get_config()
         if config.setting["browser_integration_localhost_only"]:
-            host_address = '127.0.0.1'
+            host_address = BROWSER_INTEGRATION_LOCALIP
         else:
-            host_address = '0.0.0.0'  # nosec
+            host_address = LISTEN_ALL  # nosec
```

Comment:
The `# nosec` can go here. It's well possible that the bandit tool will complain on the new line above where `'0.0.0.0'` is used. We'll see.

Reply:
I saw no complaints, after https://github.com/metabrainz/picard/pull/2656/commits/4a60bb5692d698d0299007e5610772f8529dc1ce so I guess we can just drop it.